circle.yml

```diff
@@ -12,6 +12,4 @@ test:
   pre:
     - ulimit -n 400
   override:
-    - bundle exec rspec -r rspec_junit_formatter --format progress --format RspecJunitFormatter -o $CIRCLE_TEST_REPORTS/rspec/junit.xml
```

Comment:
Oh I can see how the test results are not being stored in the proper Circle location now, to see them at the top of the output